src/common/RootProviders.tsx

```diff
@@ -37,7 +39,10 @@ export function RootProviders({
         <AuthContextProvider>
           <HttpClientContextProvider baseURL={apiBaseURL}>
             <GraphQLClientContextProvider baseURL={apiBaseURL}>
-              <OAuthContextProvider authConfig={authConfig}>
+              <OAuthContextProvider
+                authConfig={authConfig}
+                modifyAuthConfig={modifyAuthConfig}
```

Comment:
My one thought about this interface is that it would be nicer to have a single getter prop: a function that receives the action being performed + an auth result (if any) and returns an auth config. I'd use the same name, `authConfig`. 

I don't think there's any technical reason that requires us to have two separate props.

Reply:
@aecorredor I'll see what I can do - I did go down this path first - but it become a little more convoluted because internally the app config is updated with some extra props https://github.com/lifeomic/react-native-sdk/blob/11461d938c9f4a33e98dde99871a296b2d3c528c/src/hooks/useOAuthFlow.tsx#L89-L97

Reply:
here is the alternative https://github.com/lifeomic/react-native-sdk/pull/523